placement/algo/mirrored.go

```diff
@@ -80,10 +80,11 @@ func (a mirroredAlgorithm) InitialPlacement(
 	}
 
 	// NB(cw): Do not validate shards for initial placement.
-	return markAllShardsAvailable(
+	p, _, err = markAllShardsAvailable(
```

Comment:
call a.MarkAllShardsAvailable() instead?

Reply:
`a.MarkAllShardsAvailable()` doesn't allow you to override the shard cutover / cutoff fns however?

Reply:
yeah I see, then reuse a.MarkAllShardsAvailable() in other places like Add/Remove etc?

Reply:
scratch that, I just saw the latest commit, LGTM